libmamba/src/core/subdirdata.cpp

```diff
@@ -693,8 +699,8 @@ namespace mamba
                 m_name + " (check zst)",
                 m_repodata_url + ".zst",
                 "",
-                /*head_only = */ true,
-                /*ignore_failure = */ true
+                /* lhead_only = */ true,
+                /* lignore_failure = */ true
```

Comment:
Why these changes? The members of DownloadRequests are named `head_only` and `ignore_failure`, this is for preparing C++20 aggregate initialization (actually, we are missing the . in front of each member)

Reply:
C++20 aggregate initialization works for aggregate initialization, here we have a regular constructor call so nothing will change in C++20. Adding the `.` won't change anything.

The names of the parameters used in the constructor are prefix with `l`, I made it match because clang-tidy can check the comment and emit a warning when function parameters do not match the one in the comment.

Reply:
> here we have a regular constructor call so nothing will change in C++20. Adding the . won't change anything

Yeah, the idea was to remove this call in C++20, but I agree having the clang-tidy check is better.